projects/dotcms-js/src/lib/core/login.service.ts

```diff
@@ -344,16 +315,12 @@ export class LoginService {
     }
 
     /**
-     * Request and store the login as _auth list.
+     * Call the logout rest api
+     * @returns Observable<any>
      */
-    // private loadLoginAsUsersList(includeNUsers: boolean, filter: string): Observable<any> {
-    //     return this.coreWebService
-    //         .requestView({
-    //             url: `${this.urls
-    //                 .loginAsUserList}?includeUsersCount=${includeNUsers}&filter=${filter}`
-    //         })
-    //         .pluck('entity');
-    // }
+    private logOutUser(): void {
+        window.location.href = LOGOUT_URL;
```

Comment:
is not need to do this request anymore?

https://github.com/dotCMS/core-web/pull/1510/files#diff-cd0ba6e2364eca90ea6a4f00fe17b2869f58b1ab31a8abbd2a16b48674914ea4L251

Reply:
Not... SAML.